modules/UKBiobank.py

```diff
@@ -0,0 +1,201 @@
+from settings import Config
+from common.HGNCParser import GeneParser
+import sys
+import logging
+import datetime
+import opentargets.model.core as opentargets
+import opentargets.model.bioentity as bioentity
+import opentargets.model.evidence.core as evidence_core
+import opentargets.model.evidence.genetics as evidence_genetics
+import opentargets.model.evidence.linkout as evidence_linkout
+import opentargets.model.evidence.association_score as association_score
+
+__copyright__ = "Copyright 2014-2018, Open Targets"
+__credits__   = ["ChuangKee Ong"]
+__license__   = "Apache 2.0"
+__version__   = "1.2.8"
+__maintainer__= "ChuangKee Ong"
+__email__     = ["data@opentargets.org"]
+__status__    = "Production"
+
+class UKBiobank():
+    def __init__(self):
+        self.evidence_strings = list()
+        self.symbols = {}
+        self.logger = logging.getLogger(__name__)
+
+    def process_ukbiobank(self):
+        gene_parser = GeneParser()
+        gene_parser._get_hgnc_data_from_json()
+
+        self.symbols = gene_parser.genes
+        self.build_evidence()
+        self.write_evidence()
+
+    def build_evidence(self, filename=Config.UKBIOBANK_FILENAME):
+
+        now = datetime.datetime.now()
+
+        '''
+            build evidence.provenance_type object
+        '''
+        provenance_type = evidence_core.BaseProvenance_Type(
+            database=evidence_core.BaseDatabase(
+                id="UKBiobank",
+                version='2018.04',
```

Comment:
hiding constants are dangerous. could you please put at module level or so to easily check and change when it is necessary? thanks